controllers/getArticleById.controllers.js

```diff
@@ -65,9 +65,13 @@ exports.postCommentById = async (req, res, next) => {
     const { article_id } = req.params;
     const { author, body } = req.body;
 
-    const postedData = await addNewComment(article_id, author, body);
+    if (typeof author !== 'string' || typeof body !== 'string') {
+      next({ status: 400, msg: 'Incorrect Input Type' });
+    } else {
```

Comment:
ideally we would want to keep the error handling out of the controller, maybe consider moving this into the model

Reply:
done